docs/starknet/architecture/understanding-kakarot.md

```diff
@@ -188,6 +188,6 @@ sustainable.
 ## Starknet MultiVM appchains, powered by Kakarot
 
 KKRT Labs will be working closely with trusted partners such as Starkware,
-Karnot and other appchain providers to deliver EVM-compatible Starknet
+Kakarot and other appchain providers to deliver EVM-compatible Starknet
```

Comment:
this isn't a typo, Karnot is one of our devops partners

Reply:
ok, I will fix it .